app/views/AIAppActionsView.tsx

```diff
@@ -0,0 +1,54 @@
+import { RouteProp, useNavigation, useRoute } from '@react-navigation/native';
+import { NativeStackNavigationOptions, NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { useEffect } from 'react';
+
+import I18n from '../i18n';
+import SafeAreaView from '../containers/SafeAreaView';
+import * as List from '../containers/List';
+import StatusBar from '../containers/StatusBar';
+import { ChatsStackParamList } from '../stacks/types';
+import { usePerformAppAction } from '../lib/hooks/usePerformAppAction';
+
+type TNavigation = NativeStackNavigationProp<ChatsStackParamList, 'AIAppActionsView'>;
+type TRoute = RouteProp<ChatsStackParamList, 'AIAppActionsView'>;
+
+const AIAppActionsView = () => {
+	const navigation = useNavigation<TNavigation>();
+	const { aiAppActionButtons, rid } = useRoute<TRoute>().params;
+	const performInteraction = usePerformAppAction(rid);
+
+	useEffect(() => {
+		const options: NativeStackNavigationOptions = {
+			title: I18n.t('AI_actions')
+		};
+
+		navigation.setOptions(options);
+	}, [navigation]);
+
+	return (
+		<SafeAreaView>
+			<StatusBar />
+			<List.Container>
+				<List.Section>
+					<List.Separator />
+					<>
+						{aiAppActionButtons.map(button => (
```

Comment:
Can this grow too much? If so, we should make it a FlatList.
Otherwise we can keep it this way.

Reply:
I was thinking that, but I don't think any workspace will have more than a few apps for AI stuff.

But we can do a `FlatList` for a safer side, what do you think?

Reply:
Now that I said that, `List.Container` is a ScrollView already.
Adding a FlatList would actually be worse 😬